### PR TITLE
Angular17 mf ben

### PR DESCRIPTION
**Pull request by BAucoinHC**
Created build script for mf-rt

### CODE REVIEW

**Comment by linghanHC** [comment]
hpfb/sdk/post.build.js and md-rt/src/environments/ files need to be updated too. there are some placeholders in the index-en/fr files which will be replaced with app specific info in the post build script.

**Review by linghanHC — APPROVED** [review]

**Comment by BAucoinHC** [comment]
> hpfb/sdk/post.build.js and md-rt/src/environments/ files need to be updated too. there are some placeholders in the index-en/fr files which will be replaced with app specific info in the post build script.

I'm not sure if I have made the changes you had in mind but the script seems to be behaving correctly now and is ready for review again.

File: projects/mf-rt/src/environments/environment.prod-en.ts
Comment (linghanHC) [inline projects/mf-rt/src/environments/environment.prod-en.ts:1]: this is the prod url: https://health-products.canada.ca/mf-fm/en/master-file-form.html#def-top, 
serverBaseUrl should point to this server

File: projects/hpfb/sdk/src/assets/breadcrumbs/mf/breadcrumbs-en.html
Comment (linghanHC) [inline projects/hpfb/sdk/src/assets/breadcrumbs/mf/breadcrumbs-en.html:1]: <html>
<body>

"breadcrumbs": [{
--
  | "title": "Canada.ca",
  | "href": "https://www.canada.ca/en.html"
  | }, {
  | "title": "Departments and agencies",
  | "href": "https://www.canada.ca/en/government/dept.html"
  | }, {
  | "title": "Health Canada",
  | "href": "https://www.canada.ca/en/health-canada.html"
  | }, {
  | "title": "Drugs and health products",
  | "href": "https://www.canada.ca/en/health-canada/services/drugs-health-products.html"
  | }, {
  | "title": "Drug products",
  | "href": "https://www.canada.ca/en/health-canada/services/drugs-health-products/drug-products.html"
  | }, {
  | "title": "Applications and Submissions - Drug Products",
  | "href": "https://www.canada.ca/en/health-canada/services/drugs-health-products/drug-products/applications-submissions.html"
  | }],


</body>
</html>

this is the breadcrumb trails. don't see mds_index.html in it.

Comment (BAucoinHC) [reply projects/hpfb/sdk/src/assets/breadcrumbs/mf/breadcrumbs-en.html:1]: I have added a new commit to fix the serverBaseUrl and the Breadcrumbs.

**Review by linghanHC — APPROVED** [review]

**Pull request merged**